test/AlarmClockTest.cpp

```diff
@@ -149,7 +149,7 @@ TEST_F(AlarmClockTest, LongTimeout_ImmediatelyDestructed) {
    unsigned int sec = 1000;
    StopWatch sw;
    std::unique_ptr<AlarmClock<seconds>> acPtr(new AlarmClock<seconds>(sec, FakeSleep));
-   EXPECT_FALSE(acPtr->Expired());
+   EXPECT_EQ(ConvertToMicroSeconds(seconds(sec)), acPtr->SleepTimeUs());
```

Comment:
In this case, since reset is being tested, you probably want to actually use the real sleep. Otherwise, do you have a race condition between the FakeSleep and reset?